### ISSUE TITLE
When is this step used : Now use the reconfiguration command to add the second and third servers to the cluster 

### ISSUE THREAD

**Issue by VanishingPuzzle** [logcabin/logcabin#242]
When the servers are elected as leaders, when is reconfiguration done? Because still, servers are running

**Comment by ongardie** [comment]
When you bring up a LogCabin server for the first time, it's not automatically added to the cluster. It will do nothing. You can use the reconfiguration command to ask the leader of the cluster to add the new server to the cluster.

**Comment by VanishingPuzzle** [comment]
Yeah , i have 3 AWS instances running, and 1 is the leader, but there is not a break in the terminal, it keeps on looking, so is there a wait time before i run the reconfigure?

**Comment by ongardie** [comment]
I'm not sure what you mean. If there's 1 server in a cluster of size 1, it should happily lead itself, without waiting for anyone else. You should be able to run the reconfiguration as soon as all the servers involved are up and the existing cluster (of size 1, in your case) has a leader.

**Comment by VanishingPuzzle** [comment]
Yes, it elects itself the leader but I have the same issue as https://github.com/logcabin/logcabin/issues/243
